tools/bug-report/pkg/content/content.go

```diff
@@ -154,18 +140,43 @@ func GetIstiodInfo(p *Params) (map[string]string, error) {
 		return nil, fmt.Errorf("getIstiodInfo requires namespace and pod")
 	}
 	ret := make(map[string]string)
-	for _, url := range istiodURLs {
-
-		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, "discovery", fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
+	for _, url := range common.IstiodDebugURLs(p.ClusterVersion) {
+		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.DiscoveryContainerName, fmt.Sprintf(`curl localhost:8080/%s`, url), p.DryRun)
```

Comment:
curl will not work on distroless. can we replace with `pilot-discovery request GET %s`?

Reply:
done.